controllers/tempo/operator_controller.go

```diff
@@ -142,5 +143,21 @@ func (r *OperatorReconciler) pruneObjects(ctx context.Context, featureGates conf
 		}
 	}
 
+	if featureGates.GrafanaOperator {
```

Comment:
`operator_controller.go` is only for objects of the operator itself ("global" things, not per TempoStack instance). Can you move/adapt this code to `findObjectsOwnedByTempoOperator` in `tempostack_create_or_update.go`? This function looks for all objects created by a specific TempoStack instance and prunes them if they are no longer required.

Reply:
Thanks for the explanation, will move!

Reply:
I have moved the code to the correct file, could you please validate it? :)